### PR TITLE
Improve items list UI

### PR DESCRIPTION
**Pull request by ischaojie**
Changes:
1. Move Select items per page after pagination.
2. Ensure alignment.
3. Ensure actions along with the checkbox.

Preview:
image

And another piece of advice: It's better to move the actions button(view/edit/delete) to the end?

### CODE REVIEW

File: sqladmin/templates/list.html [hunk 1]
```diff
@@ -164,6 +152,18 @@ <h3 class="card-title">{{ model_view.name_plural }}</h3>
...
         </li>
       </ul>
+      <div class="dropdown text-muted">
+        <a href="#" class="btn btn-sm btn-light dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          {{ request.query_params.get("pageSize") or model_view.page_size }} / Page
+        </a>
+        <div class="dropdown-menu">
+          {% for page_size_option in model_view.page_size_options %}
+          <a class="dropdown-item" href="{{ request.url.include_query_params(pageSize=page_size_option) }}">
+            {{ page_size_option }} / Page
```
Comment (aminalaee) [inline sqladmin/templates/list.html:162]: My only comment is about the wording here. Can we do something like `Show <x> items`?
I think that would be more readable.

**Review by aminalaee — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution, The UI can be improved a lot.

File: sqladmin/templates/list.html (hunk 1 above)
Comment (ischaojie) [reply sqladmin/templates/list.html:162]: Yes, I'll add it.

**Review by aminalaee — APPROVED** [review]
Thanks for your contribution

**Pull request merged**